### DISCUSSION TITLE
make install dependency conflict

### DISCUSSION THREAD

**Discussion by smhavens** [pydantic/pydantic#4742] [Question]
Output from ```python -c "import pydantic.utils; print(pydantic.utils.version_info())"```

Traceback (most recent call last):
```File "<string>", line 1, in <module>
  File "/mnt/c/Users/sydne/OneDrive/Desktop/Classes/eecs481/p6/pydantic/pydantic/__init__.py", line 2, in <module>
    from pydantic_core import ValidationError
ModuleNotFoundError: No module named 'pydantic_core'
```

I am experiencing a dependency conflict error when running ```make install``` in my virtual environment, the specific error is:
```Collecting dirty-equals==0.5.0
  Downloading dirty_equals-0.5.0-py3-none-any.whl (24 kB)
ERROR: Cannot install importlib-metadata==4.12.0 and importlib-metadata==5.0.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested importlib-metadata==4.12.0
    The user requested importlib-metadata==5.0.0
```
I am unsure if this is something wrong locally or not, all previous steps following the [Contribution Guide](https://pydantic-docs.helpmanual.io/contributing/) worked.

**Comment by samuelcolvin** [comment]
Thanks for reporting, let's track this in #4745.

(A fix is in draft already, so should be fixed fairly soon).
